apps/asap-server/scripts/restore-algolia-index.js

```diff
@@ -0,0 +1,42 @@
+const algoliasearch = require('algoliasearch');
+const fs = require('fs');
+const StreamArray = require('stream-json/streamers/StreamArray');
+
+const main = async () => {
+  const index = algoliasearch(
+    process.env.ALGOLIA_APP_ID,
+    process.env.ALGOLIA_API_KEY,
+  ).initIndex(process.env.ALGOLIA_INDEX);
+
+  const stream = fs
+    .createReadStream(process.env.ALGOLIA_BACKUP)
+    .pipe(StreamArray.withParser());
+  let chunks = [];
+
+  await stream
+    .on('data', ({ value }) => {
+      chunks.push(value);
+      if (chunks.length === 10000) {
```

Comment:
Where does this number come from?

Reply:
oh its an arbitrary number as per algolia guidebook: https://www.algolia.com/doc/guides/sending-and-managing-data/send-and-update-your-data/how-to/sending-records-in-batches/#using-the-api